### PR TITLE
Start fleshing out README

### LINKED ISSUE #1: Flesh out README
https://drive.google.com/file/d/1xPf56h_ANASOQo4Ay8hsn_JBnRemgVa3/view?usp=sharing

### PR DESCRIPTION
**Pull request by rwwagner90**
Hopefully this is helpful. Please let me know what adjustments I should make!

Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
...
 
-* `git clone <repository-url>` this repository
-* `cd @ember/optional-features`
-* `yarn install`
+```bash
+ember install @ember/optional-features
+```
 
-## Running
+## Usage
```
Comment (chancancode) [inline README.md]: Probably should mention this is intended for toggling Ember features in _apps_ (not addons)

File: README.md [hunk 2]
```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
...
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+### List available features
```
Comment (chancancode) [inline README.md]: Newline after this

File: README.md [hunk 3]
```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
...
+Features will only be available in canary builds of ember-source. To list all available features run:
```
Comment (chancancode) [inline README.md]: > in canary builds of ember-source

This is not actually true. It is intended to work with stable builds (unlike experimental feature flags, this is to opt-in/out of things that we can't easily "just change" due to compatibility/semver concerns). However, as of now, the earliest version that has any optional features is 3.1, which is still in its canary cycle atm.

File: README.md [hunk 4]
```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
...
 
-* `ember build`
+Similarly, if you want to disable a feature, you can run:
 
-For more information on using ember-cli, visit [https://ember-cli.com/](https://ember-cli.com/).
+```bash
+ember feature:disable some-feature
+```
+
+### Installing ember-source canary
```
Comment (chancancode) [inline README.md]: @rwjblue can you confirm this is correct

File: README.md [hunk 5]
```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
```
Comment (chancancode) [inline README.md]: Probably should delete this and replace it with a one line description of what this does.

File: README.md (hunk 4 above)
Comment (rwwagner90) [reply README.md]: I spoke with @rwjblue on slack about this, and currently this is the best way. We need to fix the builds app to have correct links to canary builds, and then that will be the preferred method. I created an issue to do that work and assigned to myself https://github.com/ember-learn/builds/issues/62

File: README.md (hunk 1 above)
Comment (rwwagner90) [reply README.md]: I added this detail and a quick description in place of the original default about stuff.

**Comment by chancancode** [comment]
slack_-_emberjs_community

@rwjblue explaining the difference between this addon on the experimental feature flags

File: README.md (hunk 3 above)
Comment (chancancode) [reply README.md]: slack_-_emberjs_community

**Comment by rwwagner90** [comment]
@chancancode how does this look now?

**Comment by chancancode** [comment]
Thank you!

**Pull request merged**